### PR TITLE
spanish translation added

### PR DESCRIPTION
**Pull request by kurai021**

### CODE REVIEW

File: ES/script.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+function openFileDialog() {
+  var fileElem = document.getElementById('fileElem');
+  fileElem.click();
+}
+
+function loadPicture(evt) {
+  var file = evt.target.files[0];
+  if (!file.type.match('image.*')) {
+    alert("Oops! Looks like you didn't select an image?");
```
Comment (Ryuno-Ki) [inline ES/script.js:15]: Missed a translation, @kurai021?

File: ITA/index.html [hunk 2]
```diff
@@ -19,53 +19,48 @@
...
       });
     }
-
   </script>
 
     <h1>Naviga sicuro, proteggi la tua privacy</h1>
     <div class="container">
       <div class="manifesto">
-        " Individuals' security and privacy on the Internet are fundamental and must not be treated as optional." 
+        " Individuals' security and privacy on the Internet are fundamental and must not be treated as optional."
```
Comment (Ryuno-Ki) [inline ITA/index.html:63]: I suggest you quote https://www.mozilla.org/es-MX/about/manifesto/#principle-04

File: ITA/index.html [hunk 3]
```diff
@@ -92,6 +87,7 @@
       <a href="../" class="EN lang">EN</a>
       <a href="#" class="ITA lang">ITA</a>
       <a href="../PT-BR" class="BR lang">PT-BR</a>
+      <a href="../ES" class="ES lang">ES-BR</a>
```
Comment (Ryuno-Ki) [inline ITA/index.html:90]: Do we have an order here? If not I'd prefer alphabetically.

**Comment by Ryuno-Ki** [comment]
I've left some comments.

**Comment by shahbaz17** [comment]
LGTM 
But Yes there are few translation fix you need to do.
Can you fix this? 
Will merge as its get fixed.

**Comment by kurai021** [comment]
Hi, I'm working with the fixes

File: ITA/index.html (hunk 2 above)
Comment (kurai021) [reply ITA/index.html:63]: Hi, this is the italian translation

**Pull request closed without merging**